src/main/kotlin/org/jitsi/nlj/stats/EndpointConnectionStats.kt

```diff
@@ -85,22 +90,23 @@ class EndpointConnectionStats(
         when (packet) {
             is RtcpSrPacket -> {
                 logger.cdebug { "Tracking sent SR packet with compacted timestamp ${packet.senderInfo.compactedNtpTimestamp}" }
-                srSentTimes[Pair(packet.senderSsrc, packet.senderInfo.compactedNtpTimestamp)] =
-                        System.currentTimeMillis()
+                srSentTimes[Pair(packet.senderSsrc, packet.senderInfo.compactedNtpTimestamp)] = clock.instant()
             }
         }
     }
 
-    private fun processReportBlock(receivedTime: Long, reportBlock: RtcpReportBlock) {
+    private fun processReportBlock(receivedTime: Instant, reportBlock: RtcpReportBlock) {
         if (reportBlock.lastSrTimestamp > 0 && reportBlock.delaySinceLastSr > 0) {
```

Comment:
I'm not sure why you require both of these to be non-zero? I think zero is a legit value for each of them, especially since lastSrTimestamp is the middle 32 bits of a 64-bit value.

Are there systems that leave this zero because they're not filling it out properly, or something?

Reply:
i have a feeling that this was just a port of old logic, but don't remember for sure.  i thought, though, that it was possible to send a report block without having received an SR? and 0s in these fields denoted that scenario (or, as you point out, maybe just `delaySinceLastSr` at least?)

Reply:
Oh, yes, you're right.  Sorry.  The spec does say that the fields are set to zero if no SR has been received yet.

I think the logic is reversed -- since both fields are required to be zero if no SR has been received, it probably should be that we skip if *both* fields are zero, not if *either one* is.